fastlane/lib/fastlane/actions/build_ios_app.rb

```diff
@@ -121,7 +121,7 @@ def self.example_code
             clean: true,
             output_directory: "path/to/dir", # Destination directory. Defaults to current directory.
             output_name: "my-app.ipa",       # specify the name of the .ipa file to generate (including file extension)
-            sdk: "10.0"                      # use SDK as the name or path of the base SDK when building the project.
+            sdk: "iOS 11.1"                  # use SDK as the name or path of the base SDK when building the project.
```

Comment:
Thanks for your contribution @robmoorman, and welcome to the fastlane community 🚀  Quick question: Does it work without the `iOS ` piece, or is this part required now? Can you try just `iOS 11.1` and see if that works also?